### PR TITLE
Replaced old build process with current drone config

### PR DESCRIPTION
**Pull request by tboerger**

### CODE REVIEW

**Comment by tboerger** [comment]
~~It's failing, but to solve that we have to rewrite parts of the plugin. I would prefer to do that in a followup PR.~~

**Comment by tboerger** [comment]
This PR also migrates the 0.4 plugin to Drone >= 0.5!

File: .drone.yml [hunk 1]
```diff
@@ -1,39 +1,149 @@
...
+      - CGO_ENABLED=0
+    commands:
+      - |
+        if test "${DRONE_TAG}" = ""; then
+          go build -v -ldflags "-X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-anynines
+        else
+          go build -v -ldflags "-X main.version=${DRONE_TAG##v} -X main.build=${DRONE_BUILD_NUMBER}" -a -o release/linux/amd64/drone-anynines
+        fi
+
+  # build_linux_i386:
```
Comment (tonglil) [inline .drone.yml:37]: why are these commented out?

**Review by tonglil — APPROVED** [review]

File: .drone.yml (hunk 1 above)
Comment (tboerger) [reply .drone.yml:37]: Because it's using a ruby gem within the container, which requires an native agent to build these images.

**Pull request merged**